src/main.rs

```diff
@@ -1,3 +1,8 @@
+use clap::{App, Arg, ArgMatches};
+
 fn main() {
-    println!("Hello, world!");
+    let _matches: ArgMatches = App::new("rainyday")
+        .version("0.0.1")
```

Comment:
You can pull this from the cargo.toml like so:

`.version(crate_version!())`

[example here](https://github.com/clap-rs/clap/blob/HEAD/examples/09_auto_version.rs)

Reply:
Very cool! See 1ea28dd.